### DISCUSSION TITLE
ERROR smithay_client_toolkit::window::concept_frame No font could be found

### DISCUSSION THREAD

**Discussion by randall-coding** [hecrj/iced#865] [Q&A] [answered]
_Ubuntu 20.04, Kernel 5.8.0, Wayland, Radeon HD 7730M, installed vulkan tools using [this guide](https://linuxconfig.org/install-and-test-vulkan-on-linux)_

I was first running into a bunch of font errors as mentioned [here](https://github.com/hecrj/iced/issues/386) #386, I followed the advice to install fontconfig (and libfontconfig1-dev) and those errors went away.  However one remained.  Here is the output I see when I try to `cargo run` the **tours** example.  

> WARNING: radv is not a conformant vulkan implementation, testing use only.
> [2021-04-08T18:15:59Z ERROR smithay_client_toolkit::window::concept_frame] No font could be found

I'm not sure how do deal with either line 1 or 2.  On line one, it warns about radv.  But in my grub command I've modified it to set amdgpu on and radeon support off like so:

>GRUB_CMDLINE_LINUX_DEFAULT="radeon.si_support=0 radeon.cik_support=0 amdgpu.si_support=1 amdgpu.cik_support=1 amdgpu.dc=1 amdgpu.dpm=1 amdgpu.modeset=1"

For the second line of output (the "ERROR smithay_client_toolkit::window::concept_frame"), I have no idea what might fix that and nothing helpful showing up on search engines.

Any advice on how to fix this?

**Comment by alyti** [comment, marked as answer]
radv is mesa vulkan driver, its probably just wanting you to use amdvlk which is an official amd vulkan driver, as for the smithay error it seems to be coming from winit which depends on smithay for windowing on wayland and error was added here https://github.com/Smithay/client-toolkit/pull/168, i'd probably start by making sure your fontconfig is okay: https://github.com/alacritty/alacritty/issues/4591#issuecomment-748440633 (note i don't use linux often)

**Comment by randall-coding** [comment]
Very good advice for someone who doesn't use Linux often, thanks.
